src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -268,8 +268,11 @@ def perform_post_processing(
 
         obs_data = obs_data[list(FINAL_OBSERVERS[observer])]
         logger.info(f"Writing final results for {observer}.")
+        obs_dir = final_output_dir / observer
+        obs_dir.mkdir(parents=True, exist_ok=False)
```

Comment:
Why is `exist_ok` False? This seems highly problematic when we're parallelized, since we want all the parallel jobs to write to the same directory. Also, why are we now creating the directory at this stage? When were we creating it before? Are we still creating it at that time, because I don't see any changes that remove directory creation.

Reply:
Oh yeah, good call. 

No, we have never created final results in observer directories. It seemed worthwhile since we are now spitting out many files per observer.